01-Login/README.md

```diff
@@ -1,84 +1,705 @@
-# Scenario #1 - Logging In and Gated Content
+---
+title: Login
+description: This tutorial demonstrates how to add user login to a Vue.JS application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - vuejs
+  - login
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
 
-This sample demonstrates:
+<!-- markdownlint-disable MD034 MD041 -->
 
-- Logging in to Auth0 using Redirect Mode
-- Accessing profile information that has been provided in the ID token
-- Gated content. The `/profile` route is not accessible without having first logged in
+::: note
+**New to Auth?** Learn [How Auth0 works](/overview), how it [integrates with Single-Page Applications](/architecture-scenarios/application/spa-api) and which [protocol](/flows/concepts/implicit) it uses.
+:::
 
-## Project setup
+## Configure Auth0
+### Get Your Application Keys
+
+When you signed up for Auth0, a new application was created for you, or you could have created a new one.
+
+You will need some details about that application to communicate with Auth0. You can get these details from the [Application Settings](${manage_url}/#/applications/${account.clientId}/settings) section in the Auth0 dashboard.
```

Comment:
> [Application Settings](${manage_url}/#/applications/${account.clientId}/settings)

Substitutions like `${manage_url}` and `${account.clientid}` don't mean anything in GitHub, they only work on the docs site. These will need converted to actual links or the sentence rewritten so that the link isn't included.

For example, the `manage_url` is https://manage.auth0.com, APIs URL is https://manage.auth0.com/#/apis, etc.

In this case, I think the link will have to be removed and the sentence rewritten, as you won't have access to the client ID to take the reader directly to the application settings. Something like:

 ```suggestion
You will need some details about that application to communicate with Auth0. You can get these details from your SPA app configuration settings in the [Auth0 dashboard](https://manage.auth0.com/#/applications).
```